discovery_engine/lib/src/domain/models/embedding.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Xayn AG
```

Comment:
```suggestion
// Copyright 2022 Xayn AG
```
shouldn't matter too much I think

Reply:
@acrrd is it possible with github to only run the copyright job on modified files of the current pr? then we could change the copyright check to allow only for the current year, which will catch such copy-paste-issues.

Reply:
Not sure that will solve the problem. We can modify a file that was introduced last year so checking for current here will be wrong. We need to check for newly added files.
For the script that added the copyright to every file I used the year in which the file was introduced in git but it is more complex.

Reply:
mhh i see :/ i guess it's easier to keep it like it is and be careful in copy-pasting stuff around.